### DISCUSSION TITLE
iso.bin.enc not been loaded with webMAN

### DISCUSSION THREAD

**Discussion by mailliwal** [aldostools/webMAN-MOD#1025] [Q&A]
Hi,

I would like to play PS2 games on PS3 SuperSlim but game couldn't loaded.

1) Encrypted PS2 game ISO to ISO.bin.enc
2) Copied to NTFS USB HDD under /PS2ISO/ISO.bin.enc
3) Tested PS2 Classics Placeholder could load and user DualShock 3 controller
4) Mount PS2 game under Game > webMAN Games >PLAYSTATION2 > ISO.bin.enc
5) Error pop-up - HEN does not support ps2 iso use encrypted game

May I know anything goes wrong?

Thanks

**Comment by aldostools** [comment]
The encrypted ISO must have the extension .BIN.ENC in caps. Otherwise it will not be playable since the PS3 file system is case sensitive.

The PS2 games only are playable from /dev_hdd0/PS2ISO. webMAN tries to copy the external PS2 games to the internal HDD, but sometimes it may fail (e.g. if HDD is full or if the copy is interrupted).

The BIN.ENC must be encrypted with the universal content ID and key. If other key is used, it will require to install a RAP.

**Reply by mailliwal** [reply]
- Checked file extension is *.BIN.ENC (e.g. "Shadow of the Colossus.BIN.ENC")
- I will try to copy game to internal HDD instead of external USB HDD
- Could you share RAP file, is it in PKG format ?

Thanks

**Reply by aldostools** [reply]
The **2P0001-PS2U10000_00-0000111122223333.rap** is the default  RAP in [PS2 Classics GUI](https://www.psx-place.com/resources/ps2-classics-gui.638/)

If that RAP is used, it is created by default by PS3HEN. So it is not necessary. Or use Apollo Save Tool to install it if needed.
Or copy the rap containing the binary code "F5 DE CA BB 09 88 4F F4 02 D4 12 3C 25 01 71 D9" to /dev_usb000/exdata/2P0001-PS2U10000_00-0000111122223333.rap, Use an HexEditor to enter these HEX codes.
